test/testsuite.js

```diff
@@ -1,53 +1,51 @@
-import test from 'tape-promise/tape'
-import { randomBytes } from 'crypto'
-import fs from 'fs-extra'
-import * as secureRemove from '../src'
-import tmp from 'tempfile'
+const test = require('tape-promise/tape')
+const { randomBytes } = require('crypto')
+const fs = require('fs-extra')
+const secureRemove = require('..')
+const tmp = require('tempfile')
 
-export default function (method) {
-  test(`file not exists (${method})`, async (t) => {
+function testsuite () {
```

Comment:
No need for testsuite function; just remove it and put tests in the file scope.

Reply:
Will do. I kept that to minimize the diff for now.